scripts/deploy.js

```diff
@@ -22,9 +22,9 @@ const maxAttesters = 2 ** circuitUserStateTreeDepth - 1
     const unirep = await deployUnirep(
         signer,
         {
-            globalStateTreeDepth: 5,
-            userStateTreeDepth: 5,
-            epochTreeDepth: 32,
+            globalStateTreeDepth,
+            userStateTreeDepth,
+            epochTreeDepth,
         },
         {
             attestingFee,
```

Comment:
actually they are in the same config
`const deployUnirep = async (deployer, _settings) => {}`
so set the config together if you use `deployUnirep` 😅
e.g.
```typescript
{
    globalStateTreeDepth,
    userStateTreeDepth,
    epochTreeDepth,
    attestingFee,
    ...
}
```